response_operations_ui/views/messages.py

```diff
@@ -83,9 +84,10 @@ def view_conversation(thread_id):
                 _get_message_json(form,
                                   thread_id=refined_thread[0]['thread_id'])
             )
-            thread_url = url_for("messages_bp.view_conversation", thread_id=thread_id) + "#latest-message"
+            thread_url = url_for("messages_bp.view_conversation", thread_id=thread_id, page=page) + "#latest-message"
```

Comment:
It now returns to the first page once there has been a reply but the flash message is still returning the user to the older page that it was on.